src/ArcanaPrime.sol

```diff
@@ -396,6 +418,11 @@ contract ArcanaPrime is ERC721AQueryable, ERC721ABurnable, Ownable, OperatorFilt
         _;
     }
 
+    modifier isBelowMintSupply(uint256 _amount) {
```

Comment:
Modifier checks if "is below or equal to mint supply, otherwise revert", so the name `isBelowMintSupply` is not 100% correct - same for `isBelowMaxSupply`